the-events-calendar.php

```diff
@@ -27,11 +27,10 @@
 Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 
+define( 'THE_EVENTS_CALENDAR_DIR', dirname( __FILE__ ) );
+define( 'THE_EVENTS_CALENDAR_FILE', __FILE__ );
```

Comment:
Fwiw, I'm not too big on `__MAGIC__` style notation here, because our own constants will never be magical (unlike `__FILE__` etc, which in a sense changes value depending on where you call it from).